src/views/BusinessCase/index.tsx

```diff
@@ -13,34 +13,60 @@ import GeneralProjectInfo from './GeneralProjectInfo';
 import ProjectDescription from './ProjectDescription';
 import AsIsSolution from './AsIsSolution';
 import PreferredSolution from './PreferredSolution';
+import AlternativeSolution from './AlternativeSolution';
 import './index.scss';
 
+// Default data for estimated lifecycle costs
+const defaultEstimatedLifecycle = {
+  year1: [{ phase: '', cost: '' }],
+  year2: [{ phase: '', cost: '' }],
+  year3: [{ phase: '', cost: '' }],
+  year4: [{ phase: '', cost: '' }],
+  year5: [{ phase: '', cost: '' }]
+};
+
+// Default data for a proposed solution
+export const defaultProposedSolution = {
+  title: '',
+  summary: '',
+  acquisitionApproach: '',
+  pros: '',
+  cons: '',
+  estimatedLifecycleCost: defaultEstimatedLifecycle,
+  costSavings: ''
+};
+
 export const BusinessCase = () => {
-  const pages = [
+  const [pages, setPages] = useState([
```

Comment:
The pages are managed dynamically with react hooks since Alternative B is not included by default. It need to be added by the user.